doc/api/cli.md

```diff
@@ -288,6 +288,17 @@ Path to the file used to store the persistent REPL history. The default path is
 to an empty string (`""` or `" "`) disables persistent REPL history.
 
 
+### `NODE_TTY_UNSAFE_ASYNC=1`
+<!-- YAML
+added: REPLACEME
+-->
+
+When set to `1` writes to stdout and stderr will be non-blocking and async when
+outputting to a TTY on platforms which support async stdio.
```

Comment:
Nit: replace `outputting` with `writing` or some appropriate verb. (Using `output` as a verb rather than a noun is awkward, at least to me.)

Nit: I won't leave these specific nits everywhere but `async` is better (in my opinion, anyway) as `asynchronous` and all the stdio/stdin/stdout/stderr abbreviations should either be in backticks or else written out rather than shortened. (My opinion only, nits, you can ignore them, etc.)